app/models/issue.rb

```diff
@@ -49,4 +51,9 @@ def sub_id
   def self.find_from_project(project, sub_id)
     Issue.find(project.issue_ids[sub_id.to_i - 1])
   end
+
+  def set_sub_id
+    current_id = project.issues.count
```

Comment:
> Placing this is a before_save callback with validations in place should greatly reduce the chance of this happening. If we don't want this to happen at all

I do get your point but I don't how would we use a before_save callback for this issue, wouldn't it change the sub_id every time an issue is updated i.e., saved ?

> By the way, what OS/terminal do you use? It looks great!

I use Gnome Terminal on Fedora 21 ;)